Automattic/wp-calypso
# Gutenberg Jetpack Preset: Have Beta Blocks Availability Default to `true`

#### Changes proposed in this Pull Request

Quoting pafL3P-cY-p2:

> Part of the confusion is due to people being unsure how much they have to do on the server side when first starting to write a block. Up to now, we’ve required that block to be present in at least one of the aforementioned lists — meaning developers writing a new block cannot simply do that entirely in Calypso but will also have to write a server-side patch. I think we can make everyone’s life easier if we change the block availability default for beta blocks to true — that way, the server side patch wouldn’t be needed anymore.

#### Testing instructions

Verify that in local Calypso, you should see all beta blocks (including Mailchimp -- compare that to `master` where it's not available).
